.github/workflows/docker-builds.yml

```diff
@@ -56,6 +56,7 @@ jobs:
           - docker-image-name: pytorch-linux-focal-linter
           - docker-image-name: pytorch-linux-jammy-cuda11.8-cudnn8-py3.9-linter
           - docker-image-name: pytorch-linux-jammy-py3-clang12-executorch
+          - docker-image-name: pytorch-linux-jammy-aarch64-py3.10-gcc11
```

Comment:
To fix this docker build issue please use this approach: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations
with different runners required for different docker image then this could be passed to runs-on as matrix parameter.

Reply:
extended matrix and included aarch64 linux job